app-modules/service-management/src/Filament/Resources/ServiceRequestResource/Pages/CreateServiceRequest.php

```diff
@@ -83,7 +84,15 @@ public function form(Form $form): Form
                 Grid::make()
                     ->schema([
                         Select::make('type_id')
-                            ->options(ServiceRequestType::pluck('name', 'id'))
+                            ->options(ServiceRequestType::when(! auth()->user()->hasRole('authorization.super_admin'), function (Builder $query) {
+                                $query->whereHas('managers', function (Builder $query): void {
+                                    $query->where('teams.id', auth()->user()->teams()->first()?->getKey());
+                                })->orWhereHas('auditors', function (Builder $query): void {
+                                    $query->where('teams.id', auth()->user()->teams()->first()?->getKey());
+                                });
```

Comment:
Let's remove the auditor check here. You should only be able to create ones you are a manager of.